src/components/AltinnAppHeader.module.css

```diff
@@ -0,0 +1,95 @@
+.appBarWrapper {
+  flex-grow: 1;
+}
+
+.appBarWrapper header {
+  box-shadow: none;
+}
+
+.blueDark {
+  color: #0062ba;
```

Comment:
I think this color might be this in the designsystem (among others) `--fds-semantic-surface-action-default`. But I also see that the color value is used many places throughout the codebase :sweat_smile: